lib/update/internal/rollingupdate/builder.go

```diff
@@ -132,7 +132,8 @@ func (r Builder) restart(server storage.UpdateServer) update.Phase {
 	node := r.node("restart", "Restart container on node %q", server.Hostname)
 	node.Executor = libphase.RestartContainer
 	node.Data = &storage.OperationPhaseData{
-		Package: &r.App,
+		ExecServer: &server.Server,
```

Comment:
Inadvertently removed the link to the target server to execute the command one with the recent commit.